crates/router/src/configs/defaults.rs

```diff
@@ -509,26 +739,21 @@ impl Default for super::settings::RequiredFields {
                                     RequiredFieldInfo {
                                         required_field: "billing.address.country".to_string(),
                                         display_name: "country".to_string(),
-                                        field_type: enums::FieldType::UserCountry {
-                                            options: vec!["US".to_string(), "IN".to_string()],
+                                        field_type: enums::FieldType::UserAddressCountry {
+                                            options: vec!["ALL".to_string()],
                                         },
                                     },
                                     RequiredFieldInfo {
                                         required_field: "email".to_string(),
                                         display_name: "email".to_string(),
                                         field_type: enums::FieldType::UserEmailAddress,
                                     },
-                                    RequiredFieldInfo {
-                                        required_field: "browser_info".to_string(),
-                                        display_name: "browser_info".to_string(),
-                                        field_type: enums::FieldType::Text,
-                                    },
                                 ],
                             ),
                             (
                                 enums::Connector::Worldline,
                                 vec![RequiredFieldInfo {
-                                    required_field: "card.card_holder_name".to_string(),
+                                    required_field: "payment_method_datacard.card_holder_name".to_string(),
```

Comment:
Was this supposed to be:

```suggestion
                                    required_field: "payment_method_data.card.card_holder_name".to_string(),
```

Reply:
yes, missed this, will update now